app/src/main/java/ch/epfl/cs311/wanderwave/viewmodel/ProfileViewModel.kt

```diff
@@ -163,6 +163,7 @@ constructor(
     viewModelScope.launch {
       repository.getItem(id).collect { fetchedProfile ->
         fetchedProfile.onSuccess { profile ->
+          profile.profilePictureUri = null // TODO : @Clarence, pls can you do it more properly
```

Comment:
As decided on discord, with regards to the PRD, and the "future improvements" we can do, I don't think we should remove this. Just leave it as null.